lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -403,7 +403,8 @@ def get_javascript(cls, stream=None):
         for filetype, ext in sorted(FigureCanvasWebAggCore.
                                     get_supported_filetypes_grouped().
                                     items()):
-            extensions.append(ext[0])
+            if not ext[0] == 'pgf':  # pgf does not support BytesIO
```

Comment:
Or are you saying that the backend currently doesn't work with BytesIO instances at all? If so, we should open a new issue and I will look into that.